api/admin/controller.py

```diff
@@ -983,6 +982,7 @@ def lanes_for_parent(parent):
             for list in lane.customlists:
                 if list.id not in custom_list_ids:
                     lane.customlists.remove(list)
+            lane.update_size(self._db)
```

Comment:
This ensures that the count is updated whenever the lane is edited, without having to wait for the materialized view script to run again.

Reply:
One other place you could do it is when a custom list is edited, if the list is associated with any lanes.

Reply:
Good idea. This is ready for another look.